src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+
+
+@validataclass
+class OpenDataSwissCapacitiesInput:
+    categoryType: OpenDataSwissCapacityCategoryTypeInput = EnumValidator(OpenDataSwissCapacityCategoryTypeInput)
+    total: int = IntegerValidator()
+
+
+@validataclass
+class OpenDataSwissAdditionalInformationInput:
+    de: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    en: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    it: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    fr: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+
+
+class OpenDataSwissOperationTimeDaysOfWeek(Enum):
+    MONDAY = 'MONDAY'
+    TUESDAY = 'TUESDAY'
+    WEDNESDAY = 'WEDNESDAY'
+    THURSDAY = 'THURSDAY'
+    FRIDAY = 'FRIDAY'
+    SATURDAY = 'SATURDAY'
+    SUNDAY = 'SUNDAY'
+
+    def to_osm_opening_day_format(self) -> str:
+        return {
+            self.MONDAY: 'Mo',
+            self.TUESDAY: 'Tu',
+            self.WEDNESDAY: 'We',
+            self.THURSDAY: 'Th',
+            self.FRIDAY: 'Fr',
+            self.SATURDAY: 'Sa',
+            self.SUNDAY: 'Su',
+        }.get(self, None)
+
+
+@validataclass
+class OpenDataSwissOperationTimeInput:
+    operatingFrom: Optional[str] = Noneable(StringValidator())
+    operatingTo: Optional[str] = Noneable(StringValidator())
+    daysOfWeek: Optional[list[str]] = Noneable(ListValidator(EnumValidator(OpenDataSwissOperationTimeDaysOfWeek)))
+
+
+@validataclass
+class OpenDataSwissPropertiesInput:
+    operator: str = StringValidator()
+    displayName: str = StringValidator()
+    address: Optional[OpenDataSwissAddressInput] = Noneable(DataclassValidator(OpenDataSwissAddressInput))
+    capacities: list[OpenDataSwissCapacitiesInput] = ListValidator(DataclassValidator(OpenDataSwissCapacitiesInput))
+    additionalInformationForCustomers: Optional[OpenDataSwissAdditionalInformationInput] = Noneable(
+        DataclassValidator(OpenDataSwissAdditionalInformationInput)
+    )
+    parkingFacilityCategory: str = StringValidator()
+    parkingFacilityType: Optional[str] = Noneable(StringValidator())
+    salesChannels: Optional[list[str]] = Noneable(ListValidator(StringValidator()))
+    operationTime: Optional[OpenDataSwissOperationTimeInput] = Noneable(DataclassValidator(OpenDataSwissOperationTimeInput))
+
+    def __post_init__(self):
+        for capacity in self.capacities:
+            if capacity.categoryType == OpenDataSwissCapacityCategoryTypeInput.STANDARD:
+                return
+        # If no capacity with type PARKING was found, we miss the capacity and therefore throw a validation error
+        raise ValidationError(reason='Missing parking spaces capacity')
+
+    def get_osm_opening_hours(self) -> str:
+        open_swiss_opening_times_by_weekday: dict[OpenDataSwissOperationTimeDaysOfWeek, list[str]] = defaultdict(list)
+        check_counter_24_7: int = 0
+        check_list_weekday: list[str] = []
```

Comment:
Why collecting these at all, if the times are always the same by format definition?